src/vcpkg/export.prefab.cpp

```diff
@@ -281,8 +282,12 @@ namespace vcpkg::Prefab
 
         for (const auto& triplet_file : triplet_db.available_triplets)
         {
-            if (triplet_file.name.size() > 0)
+            if (!triplet_file.name.empty())
             {
+                // The execution of emscripten cmake script causes the prefab export to fail.
+                // But here we don't need this execution at all, so we skip it.
+                if (triplet_file.name == "wasm32-emscripten") continue;
```

Comment:
IMO this isn't the right direction. Triplet names don't carry much meaning. They are identifiers.

Basically, it shouldn't care about any triplet which isn't installed.
Looking at the past issues, I think what is really desired is a user provided positive list of triplets, so that you can pick some architectures.

Reply:
The fix is quite simple, yet it will repair the prefab export, which has been broken for 4 (?) years.

When I think about the command line, I would just omit the triplets entirely.
``` bash
vcpkg export prefab
```
or or if you just want to export a specific ABI 

``` bash
vcpkg export prefab  --ABI arm64-v8a
```

Reply:
> which has been broken for 4 (?) years.

A broken fix is not made correct because the bug attempting to be fixed is old.

I agree with @dg0yt that you can't assume anything based on the triplet name. I can make my identical-to-wasm32-emscripten triplet named "fish".